seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/connector-redis-e2e-redis3/src/test/java/org/apache/seatunnel/e2e/connector/redis/Redis3IT.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seatunnel.e2e.connector.redis;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Redis3IT extends RedisTestCaseTemplateIT {
```

Comment:
I think we can just put all IT like `Redis3IT`, `Redis4IT` into same moudle.

Reply:
> I think we can just put all IT like `Redis3IT`, `Redis4IT` into same moudle.

The initial version was like this, but CI would run out of time

Reply:
So I didn't think of a great way to run so many versions of tests and ensure that CI runs without timeout